content/en/docs/reference/command-line-tools-reference/feature-gates/kubelet-ensure-secret-pulled-images.md

```diff
@@ -0,0 +1,15 @@
+---
+title: KubeletEnsureSecretPulledImages
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.33"
+---
+Forces the Kubelet to re-verify image pull credentials
+for pre-puled images.
```

Comment:
A slightly more detailed description, please. Also, as I mentioned in an earlier comment - it's not just about pre-pulled images.